playroom/snippets.tsx

```diff
@@ -8,44 +8,45 @@ const menuSnippet = {
     name: 'Menu',
     code: `
     <Menu
-      width={400}
       renderTarget={({ ref, onPress, isMenuOpen }) => (
-        <Touchable ref={ref} onPress={onPress} style={{ width: 100 }}>
-          <Inline space={16}>
-            <IconKebabMenuLight />
-            <Text3 regular>{isMenuOpen ? "Close" : "Open"}</Text3>
-          </Inline>
-        </Touchable>
+        <Box padding={16}>
+          <Touchable
+            ref={ref}
+            onPress={onPress}
+            style={{ maxWidth: "fit-content" }}
```

Comment:
why this inline style? 🤔

Reply:
Discussed offline, it's because we don't have css classes in playroom